test/e2e/deprecated_e2e_test.go

```diff
@@ -18,12 +21,25 @@ import (
 var missingAPI = `{"apiVersion":"verticalpodautoscalers.autoscaling.k8s.io/v1","kind":"VerticalPodAutoscaler","metadata":{"name":"my.thing","namespace":"foo"}}`
 
 var _ = Describe("Not found APIs", func() {
+
+	var ns corev1.Namespace
+
 	BeforeEach(func() {
-		csv := newCSV("test-csv", testNamespace, "", semver.Version{}, nil, nil, nil)
+		namespaceName := genName("catsrc-e2e-")
```

Comment:
nit: strange prefix.  It may be better some string that can be used to identify this test case like "dep-e2e"

Reply:
Yup +1

Reply:
Looks like this was addressed - resolving the conversation.

Reply:
copy/pasta bug =D great catch! thank you =D